Library/Formula/ffts.rb

```diff
@@ -2,8 +2,9 @@ class Ffts < Formula
   desc "C library that computes the discrete Fourier transform"
   homepage "http://anthonix.com/ffts/"
   head "https://github.com/anthonix/ffts.git"
-  url "http://anthonix.com/ffts/releases/ffts-0.7.tar.gz"
-  sha256 "6362e498e5aa241661cfe9060153076b78e300a5d3c365997ba6ac2f637df3ff"
+  url "https://github.com/anthonix/ffts/archive/fa1780c68593762b1e4bdbc46d83912db3eba27a.tar.gz"
```

Comment:
Sorry -- that's all I got. Code matches, my tests pass. Ideally the sha256 would too, but there are additional non-source files in the GitHub tarball.